examples/defer-stream/src/index.ts

```diff
@@ -1,4 +1,4 @@
-import { yoga } from './yoga.js'
+import { yoga } from './yoga'
```

Comment:
```suggestion
import { yoga } from './yoga.js'
```

Reply:
It fails on `start` :( Maybe it is ts-node doesn't support it